examples/moses-pln-synergy/scm/README.md

```diff
@@ -164,6 +179,10 @@ $5 = (SetLink
    )
 )
 
+### (5) Build the tautology that if X takes treatment-1, then
+    treatment-1 contains compound-A. Semi-formally
+    (\x take(x, treatment-1)) -> (\x contain(treatment-1, compound-A))
```

Comment:
As long as the variables do not have overlapping scopes I don't understand what is confusing. Do you find confusing to write
```c
int inc(int x) { return x + 1; }
int dec(int x) { return x - 1; }
```
?